scripts/generate_builtins.py

```diff
@@ -29,6 +29,8 @@ def trimext(file: str) -> str:
     "filesha512": "filesha512(\"hello.txt\")",
     "list": "list(1, 2, 3)",
     "map": "map(\"a\", \"b\", \"c\", \"d\")",
+    "jsonencode": "jsonencode({ \"foo\": \"bar\", Content: \"capitalized\", \"Quoted\": \"quoted\" })",
```

Comment:
Please don't abuse this automated broad test for testing specific things. This should have been written as a new specific test (jsonencode_preserve_casing or similar)

Reply:
That is a better idea, it also would have been easier than digging into why generate_builtins was failing